frontend/javascripts/test/reducers/volumetracing_reducer.spec.js

```diff
@@ -240,46 +238,36 @@ test("VolumeTracing should not allow to set trace tool if getRequestLogZoomStep(
   const newState = VolumeTracingReducer(alteredState, setToolAction);
 
   t.is(alteredState, newState);
-  getVolumeTracingOrFail(newState.tracing).map(tracing => {
-    // Tool should not have changed
-    t.is(tracing.activeTool, VolumeToolEnum.MOVE);
-  });
+
+  // Tool should not have changed
+  t.is(newState.tracing.activeTool, AnnotationToolEnum.MOVE);
 });
 
 test("VolumeTracing should cycle trace/view/brush tool", t => {
   const cycleToolAction = VolumeTracingActions.cycleToolAction();
 
   // Cycle tool to Brush
   let newState = VolumeTracingReducer(initialState, cycleToolAction);
```

Comment:
I would expect two cycling actions to be needed now with the new skeleton tool in between move and brush.

Reply:
on this branch, the test fails indeed, but in the other PR I took care that the cycle-tool functionality only cycles through the enabled/available tools. in a volume annotation, the skeleton tool won't be available, which is why the test works again.